app/controllers/audits_controller.rb

```diff
@@ -12,6 +12,13 @@ def show
 
   def save
     audit.user = current_user
+    # Get the next_item before updating the audit. As the
```

Comment:
> Can you please add a test to drive this fix?

I've now added a test that covers moving to the next item to audit when save is clicked.